docs/usage/install/ai/get-started-macvlan.md

```diff
@@ -142,7 +142,7 @@ The network planning for the cluster is as follows:
 
 ## Install Spiderpool
 
-1. Use Helm to install Spiderpool and enable the SR-IOV component:
+1. Use Helm to install Spiderpool and enable the rdmaSharedDevicePlugin component:
```

Comment:
```suggestion
1. Use Helm to install Spiderpool and enable the rdmaSharedDevicePlugin:
```
`component` is not very necessary, and may cause confuse with Plugin.

Reply:
done